e2e/specs/drug-orders.spec.ts

```diff
@@ -222,6 +222,140 @@ test('Record, edit and discontinue a drug order', async ({ page }) => {
+  });
+
+  await test.step('When I set the dose to "1" tablet', async () => {
+    await page.getByLabel(/^dose$/i, { exact: true }).click();
+    await page.getByLabel(/^dose$/i, { exact: true }).fill('1');
+  });
+
+  await test.step('And I set the route to "Oral"', async () => {
+    await page.getByRole('combobox', { name: /route/i }).click();
+    await page.getByRole('option', { name: /oral/i }).click();
+  });
+
+  await test.step('And I set the frequency to "Once daily"', async () => {
+    await page.getByRole('combobox', { name: /frequency/i }).click();
+    await page.getByRole('option', { name: /once daily/i }).click();
+  });
+
+  await test.step('And I set duration to "3" days', async () => {
+    await page.getByText(/^duration$/i).clear();
+    await page.getByText(/^duration$/i).fill('3');
+  });
+
+  await test.step('And I set the quantity to dispense to 3', async () => {
+    await page.getByText(/^quantity to dispense$/i).clear();
+    await page.getByText(/^quantity to dispense$/i).fill('3');
+  });
+
+  await test.step('And I set the prescription refills to 1', async () => {
+    await page.getByText(/^prescription refills$/i).clear();
+    await page.getByText(/^prescription refills$/i).fill('1');
+  });
+
+  await test.step('And I set the indication to "Headache"', async () => {
+    await page.getByText(/indication/i).fill('Headache');
+  });
+
+  await test.step('And I click on the "Save Order" button', async () => {
+    await page.getByRole('button', { name: /save order/i }).click();
+  });
+
+  await test.step('Then the order status should be changed to "New"', async () => {
+    await expect(orderBasket.getByText(/incomplete/i)).toBeHidden();
+    await expect(orderBasket.getByText(/new/i)).toBeVisible();
+  });
+
+  await test.step('When I click on the "Sign and close" button', async () => {
+    await page.getByRole('button', { name: /sign and close/i }).click();
+  });
+
+  await test.step('Then I should see a success notification', async () => {
+    await expect(page.getByText(/placed order for aspirin/i)).toBeVisible();
+  });
+
+  await test.step('And I should see the newly added order in the active medications list', async () => {
+    const headerRow = medicationsPage.medicationsTable().locator('thead > tr');
+    const dataRow = medicationsPage.medicationsTable().locator('tbody > tr');
+
+    await expect(headerRow).toContainText(/start date/i);
+    await expect(headerRow).toContainText(/details/i);
+    await expect(dataRow).toContainText(/aspirin 81mg/i);
+    await expect(dataRow).toContainText(/1 tablet/i);
+    await expect(dataRow).toContainText(/oral/i);
+    await expect(dataRow).toContainText(/once daily/i);
+    await expect(dataRow).toContainText(/3 days/i);
+    await expect(dataRow).toContainText(/indication headache/i);
+  });
+
+  await test.step('When I click on the Orders section', async () => {
+    await ordersPage.page.getByRole('link', { name: 'Orders' }).click();
```

Comment:
Ditto here. I don't think we need to create the `OrdersPage` instance unless we actually need some functionality there, but everything looks like it could be re-written in terms of `page` with no functionality lost?